app/src/server/emails/sendPaymentFailed.ts

```diff
@@ -0,0 +1,25 @@
+import { env } from "~/env.mjs";
```

Comment:
We should also have an email draft for when the payment went through successfully.

Reply:
Will take a look at it.

Reply:
Added payment successful emails.